src/calendar/day/basic/index.js

```diff
@@ -92,26 +93,29 @@ class Day extends Component {
     }
 
     return (
-      <TouchableOpacity
-        testID={this.props.testID}
-        style={containerStyle}
-        onPress={this.onDayPress}
-        onLongPress={this.onDayLongPress}
-        activeOpacity={activeOpacity}
-        disabled={shouldDisableTouchEvent}
-        accessibilityRole={isDisabled ? undefined : 'button'}
-        accessibilityLabel={this.props.accessibilityLabel}
-      >
-        <Text allowFontScaling={false} style={textStyle}>{String(this.props.children)}</Text>
-        <Dot
-          theme={theme}
-          isMarked={marked}
-          dotColor={dotColor}
-          isSelected={selected}
-          isToday={isToday}
-          isDisabled={isDisabled}
-        />
-      </TouchableOpacity>
+      <View style={this.style.dayContainer}>
+        <TouchableOpacity
+          testID={this.props.testID}
+          style={containerStyle}
+          onPress={this.onDayPress}
+          onLongPress={this.onDayLongPress}
+          activeOpacity={activeOpacity}
+          disabled={shouldDisableTouchEvent}
+          accessibilityRole={isDisabled ? undefined : 'button'}
+          accessibilityLabel={this.props.accessibilityLabel}
+        >
+          <Text allowFontScaling={false} style={textStyle}>{String(this.props.children)}</Text>
+          <Dot
+            theme={theme}
+            isMarked={marked}
+            dotColor={dotColor}
+            isSelected={selected}
+            isToday={isToday}
+            isDisabled={isDisabled}
+          />
+        </TouchableOpacity>
+        {!!renderUnderView && renderUnderView()}
```

Comment:
I think it would make sense to pass `this.props.date` to `renderUnderView`. In that way it would be possible to have one `renderUnderView` function in the screen instead of creating a new function every single time.

Reply:
You meaning put the renderUnderView to the <Calendar /> props?

I also think that was a good way, but I'm worry a big changes that unable to merge with master.

If you are agreed, I would like to change it. 😂

Reply:
Yes, you could move `renderUnderView` to base Calendar props (maybe it would be worth renaming the prop to `renderUnderDayView` or something similar in that case), and then call the prop with day object like this:
`{!!renderUnderDayView && renderUnderDayView(this.props.date)}

Reply:
Ok, I know your meaning.

And I've totally changed the function as above.

Demo snapshot as below:
![image](https://user-images.githubusercontent.com/6138872/90236625-9749bd00-de55-11ea-8a3c-afa65c9a144c.png)